templates/default/yarn-site.xml.erb

```diff
@@ -1,9 +1,15 @@
 <?xml version="1.0"?>
 <?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
 <configuration>
+<property>
+    <name>yarn.resourcemanager.bind-host</name>
+    <value><%= node['fqdn'] %></value>
```

Comment:
This is not going to work if you take a snapshot of a VM for the AMI.

Reply:
Fixed. Binding address defaults to `0.0.0.0` now. It can be overwritten with the attribute `install/bind_services_private_ip` and Conda helper function `conda_helpers.bind_services_private_ip()` If set to `true`, then it will bind to the private IP.